datafusion/core/tests/memory_limit/mod.rs

```diff
@@ -180,6 +180,26 @@ async fn merge_join() {
         ])
         .with_memory_limit(1_000)
         .with_config(config)
+        .with_scenario(Scenario::AccessLogStreaming)
+        .run()
+        .await
+}
+
+#[tokio::test]
+async fn sort_merge_join_spill() {
```

Comment:
This test case can only make sure the query can run, it may or may not be spilling.

We should have some ways to verify the spilling is actually happened.

Reply:
unfortunately exactly this test case we cannot access any spilling metrics, but there is another test above `sort_merge_join_no_spill` which is exactly the same but expectedly fails by mem issue and have the spilling disabled explicitly. This test passes without issues and with spilling enabled so we can conclude the spilling happened.